Tasks/DownloadPackageV0/download.ts

```diff
@@ -13,7 +13,7 @@ import { WebApi } from 'azure-devops-node-api';
 tl.setResourcePath(path.join(__dirname, 'task.json'));
 
 async function main(): Promise<void> {
-	tl.warning("This task is deprecated. Builds that use it will break on 11-27-2023. Please switch to using DownloadPackage@1 as soon as possible.");
+	tl.warning("This task is deprecated. Functionality will stop on 01-16-2024. Please switch to using DownloadPackage@1 as soon as possible.");
```

Comment:
Why did we add a loc string but then manually warn with a different string here?